ResourceStatusSystem/Agent/EmailAgent.py

```diff
@@ -0,0 +1,148 @@
+''' EmailAgent
+
+  This agent reads a cache file ( cache.json ) which contains the aggregated information
+  of what happened to the elements of each site. After reading the cache file
+  ( by default every 30 minutes ) it sends an email for every site and then clears it.
+
+'''
+
+
+import os
+import json
+from datetime import datetime
+from DIRAC                                                       import S_OK, S_ERROR
+from DIRAC.Core.Base.AgentModule                                 import AgentModule
+from DIRAC.ResourceStatusSystem.Utilities                        import RssConfiguration
+from DIRAC.Interfaces.API.DiracAdmin                             import DiracAdmin
+
+__RCSID__ = '$Id: $'
+
+AGENT_NAME = 'ResourceStatus/EmailAgent'
+
+class EmailAgent( AgentModule ):
+
+  def __init__( self, *args, **kwargs ):
+
+    AgentModule.__init__( self, *args, **kwargs )
+    self.diracAdmin = DiracAdmin()
+
+    self.default_value = None
+    self.dirac_path = None
+    self.dirac_path = None
+    self.cacheFile = None
+
+  def initialize( self ):
+    ''' EmailAgent initialization
+    '''
+
+    self.dirac_path = os.getenv('DIRAC')
+    self.cacheFile = self.dirac_path + 'work/ResourceStatus/' + 'cache.json'
```

Comment:
Minor thing: you don't need `dirac_path` to be an object variable, since it's only used in the `initialize`. It's also repeated twice in `__init__`.